build.gradle.kts

```diff
@@ -77,6 +83,23 @@ dependencies {
     testImplementation(Wiremock.standalone)
 
     testRuntimeOnly(Junit5.engine)
+
+    // Soap stuff
+    implementation("javax.xml.ws:jaxws-api:2.3.1")
+    implementation("com.sun.xml.ws:jaxws-tools:2.3.0.2")
+
+    implementation(tjenestespesifikasjon("behandleArbeidOgAktivitetOppgave-v1-tjenestespesifikasjon"))
+    implementation(tjenestespesifikasjon("ytelseskontrakt-v3-tjenestespesifikasjon"))
+
+    implementation("org.apache.cxf:cxf-rt-features-logging:$cxfVersion")
+    implementation("org.apache.cxf:cxf-rt-frontend-jaxws:$cxfVersion")
+    implementation("org.apache.cxf:cxf-rt-ws-policy:$cxfVersion")
+    implementation("org.apache.cxf:cxf-rt-ws-security:$cxfVersion")
+    implementation("org.apache.cxf:cxf-rt-transports-http:$cxfVersion")
+    implementation("javax.activation:activation:1.1.1")
+    implementation("no.nav.helse:cxf-prometheus-metrics:dd7d125")
+    testImplementation("org.apache.cxf:cxf-rt-transports-http:$cxfVersion")
```

Comment:
Dere mangler 

```
tasks.withType<ShadowJar> {
    mergeServiceFiles()

    // Make sure the cxf service files are handled correctly so that the SOAP services work.
    // Ref https://stackoverflow.com/questions/45005287/serviceconstructionexception-when-creating-a-cxf-web-service-client-scalajava
    transform(ServiceFileTransformer::class.java) {
        setPath("META-INF/cxf")
        include("bus-extensions.txt")
    }
}
```

Reply:
fikset